test/test.make

```diff
@@ -110,7 +110,7 @@ space:= $(empty) $(empty)
 RUN_E2E = KUBECONFIG=`pwd`/_work/$(CLUSTER)/kube.config \
 	REPO_ROOT=`pwd` \
 	CLUSTER=$(CLUSTER) \
-	go test -count=1 -timeout 0 -v ./test/e2e -ginkgo.skip='$(subst $(space),|,$(TEST_E22_SKIP))'
+	go test -count=1 -parallel 1 -timeout 0 -v ./test/e2e -ginkgo.skip='$(subst $(space),|,$(TEST_E22_SKIP))'
```

Comment:
It doesn't hurt, but was it needed?

We only have a single Test* function inside the E2E binary, so it shouldn't matter, because "go test" runs those functions in parallel.

A commit message with a "why" explanation would have helped...

Reply:
I myself was not clear if this really needed :) This change was just to make sure the test failures were not caused by running in parallel.

Reply:
Then drop the commit. Without an explanation it just looks odd.